public/javascripts/SVValidate/mobile/mobileValidate.js

```diff
@@ -104,4 +104,4 @@ function resizeMobileValidation() {
     // document.getElementById("validation-not-sure-button-0").style.width = w/5-30 + "px";
     // document.getElementById("left-column-jump-button").style.left = w - 600 + "px";
     // document.getElementById("info-button").style.left = w - 700 + "px";
-}
+}
```

Comment:
Make sure to check what changes you are committing to each file with `git diff <filename>` before including them. This is a change that we don't want.

Reply:
Got it

Reply:
Wait to clarify, what is the change here? I'm looking at my code, and I don't see any difference. Is this a line ending issue or something?

Reply:
Yeah the change is that you (or your editor) removed the newline at the end of the file. The convention is to have a newline at the end of every file.

Reply:
ok, sounds good